pinot-query-runtime/src/test/resources/queries/LexicalStructure.json

```diff
@@ -135,11 +135,18 @@
         "inputs": [["1"]]
       }
     },
-    "queries": [{
-      "psql": "4.1.2.6",
-      "description": "numeric constant supports",
-      "sql": "SELECT data, 42, 3.5, 4., .001, 5e2, 1.925e-3 FROM {tbl}"
-    }]
+    "queries": [
+      {
+        "psql": "4.1.2.6",
+        "description": "numeric constant supports",
+        "sql": "SELECT data, 42, 3.5, 4., .001, 5e2, 1.925e-3 FROM {tbl}"
```

Comment:
can we add some math functions for these as well? (constant/constant and constant/column)?

Reply:
got it. will add. i wasn't sure if there's anything new to add on top of the one already there. but will do. (it also tests the RelDataTypeFactory's derive type from `+, -, *, /` methods

Reply:
test added